Libraries/Image/Image.ios.js

```diff
@@ -34,7 +34,7 @@ const ImageViewManager = NativeModules.ImageViewManager;
  * including network images, static resources, temporary local images, and
  * images from local disk, such as the camera roll.
  *
- * This exmaples shows both fetching and displaying an image from local storage as well as on from
+ * This examples shows both fetching and displaying an image from local storage as well as on from
```

Comment:
You fixed 1/2. Can you fix the other 1/2? :)

This should say "This example"  -- corrected the misspelling, but it being plural was also wrong.